Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -924,6 +924,12 @@ RefPtr<ImageBuffer> WebChromeClient::createImageBuffer(const FloatSize& size, Re
 
     return m_page.ensureRemoteRenderingBackendProxy().createImageBuffer(size, renderingMode, purpose, resolutionScale, colorSpace, pixelFormat);
 }
+
+void WebChromeClient::blockIOSurfaceCreationForNextImageBuffer()
+{
+    m_page.ensureRemoteRenderingBackendProxy().blockIOSurfaceCreationForNextImageBufferForTesting();
```

Comment:
This needs some kind of `WebProcess::singleton().shouldUseRemoteRenderingFor(purpose)` check so that the test gives sensible behavior when "Canvas in GPU process" is disabled.

Reply:
Fixed.